### PR TITLE
Add daily new cases, rename additional metrics, move vaccinations lower

### PR DESCRIPTION
**Pull request by smcclure17**
Addresses https://trello.com/c/jNylgmzc/607-more-explicitly-highlight-older-transmission-metrics

### CODE REVIEW

File: src/common/models/Projections.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export class Projections {
       Metric.VACCINATIONS,
       Metric.CASE_GROWTH_RATE,
       Metric.POSITIVE_TESTS,
+      Metric.CASE_DENSITY,
```
Comment (smcclure17) [inline src/common/models/Projections.ts:73]: To create summary data for daily new cases chart tab

File: src/components/Explore/utils.ts [hunk 2]
```diff
@@ -452,6 +457,21 @@ export const exploreMetricData: {
...
+  [ExploreMetric.DAILY_CASES_PER_100K]: {
+    title: 'Daily New Cases per 100k',
+    name: 'Daily New Cases per 100k',
+    chartId: 'daily_new_cases',
+    dataMeasure: DataMeasure.INTEGER,
+    yAxisDecimalPlaces: 0,
+    seriesList: [
+      {
+        label: 'Daily New Cases per 100k',
+        tooltipLabel: 'Daily New Cases per 100k',
```
Comment (mikelehen) [inline src/components/Explore/utils.ts:469]: nit: I think for consistency with other metrics these should be lowercase other than the first letter.

**Review by mikelehen — APPROVED** [review]
Looks good!  Don't forget we'll need to regenerate the summaries file before shipping this.

**Comment by smcclure17** [comment]
🙏 good point 

I'll merge to develop > create new snapshot > merge both changes to main

**Pull request merged**